terroroftinytown/tracker/base.py

```diff
@@ -16,10 +16,11 @@ def get_current_user(self):
         if username_raw and token:
             username = username_raw.decode('ascii')
 
-            if username:
-                return User.check_account_session(username, token)
+            if username and User.check_account_session(username, token):
+                return username
 
     def prepare(self):
+        self.application.log_filter.username = self.get_current_user()
```

Comment:
For example, we could add a function to the base handler so we can call `self.log_info('asdfasdf')` which prefixes the current username to the message.